buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -83,7 +83,7 @@ SPECS = ['aix_ppc-64_cmprssptrs'      : CURRENT_RELEASES,
          'linux_ppc-64_cmprssptrs_le' : CURRENT_RELEASES,
          'linux_x86-64'               : CURRENT_RELEASES,
          'linux_x86-64_cmprssptrs'    : CURRENT_RELEASES,
-         'linux_x86-64_cmprssptrs_cmake' : ['9'],
+         'linux_x86-64_cmprssptrs_cmake' : ['11'],
```

Comment:
Perhaps this should be `'8', '11'` now that `cmake` builds work for JDK8 and JDK11?

Reply:
It will take another ~45 min of xlinux machine to to build 8. Not sure if it is a good idea or not, but I've added for now. We can always remove it again later if problematic.

Reply:
That sounds fair.